tools/licenses/lib/main.dart

```diff
@@ -929,7 +929,7 @@ class _RepositoryDirectory extends _RepositoryEntry implements LicenseSource {
             entry.name != 'tests' &&
             entry.name != 'javatests' &&
             entry.name != 'testing' &&
-            // The directory that containts end to end tests.
+            // The directory that contains end to end tests.
             // Shoul be excluded from the licence checks.
```

Comment:
I wonder why this `Shoul` wasn't caught

Reply:
Interesting. In another spot it caught the same typo and wanted to change it to Could :(

Reply:
Fixed.